packages/deployment/test/test_lambda.js

```diff
@@ -62,7 +62,7 @@ test.serial('zipLambda: works for lambda not using message adapter', async (t) =
   t.context.lambda.useMessageAdapter = false;
   const lambdaLocalOrigin = t.context.lambda.local;
   const lambdaRemoteOrigin = t.context.lambda.remote;
-  const l = new Lambda(t.context.config)
+  const l = new Lambda(t.context.config);
```

Comment:
Could this be given a more descriptive variable name?

Reply:
Hmm, I don't think it's possible to have a *less* descriptive variable name.   How about test_lambda?

Reply:
Or testLambda,  rather.